test/multiverse/suites/redis/redis_instrumentation_test.rb

```diff
@@ -144,7 +144,7 @@ def test_does_not_record_statement_on_individual_command_node_by_default
       refute get_node[:statement]
     end
 
-    def test_records_metrics_for_set_in_web_transaction
+    def test_records_metrics_for_get_in_web_transaction
```

Comment:
Wow, good eye, thats odd that these two test names seemed to be swapped based on what the actual test does!